### ISSUE TITLE
Question about SoccerHelp back-ups

### ISSUE THREAD

**Issue by david-huddleston** [soccerhelp/soccerhelp#159]
Dustin, I remember you talking about the Soccerhelp back ups.

Don't turn in the old servers until we have good back-ups.

Did you configure back-ups on the 2 new servers/ (The is an article about how to do it in cpanel at https://www.knownhost.com/wiki/control-panels/cpanel-whm/backup-configuration ) 

Are we still going to have a back-up system on AWS so we aren't totally dependent on KnownHost? 

Call me about this - there is so much investment in SoccerHelp that it is important to have a back-up separate from the KnownHost server and on a different server - that is what all the experts say.

If we don't have something in place, let's look for a service.

2 Options >

(a). iDrive - $100 a year for 250 GB - info at https://www.idrive.com/pricing 

(b).  dropMySite.com -  30 GB of unlimited websites for $80 per year or 60 GB for $150. Read about it at https://www.dropmysite.com/ Pricing is at at https://www.dropmysite.com/en/plans 

How many GB is SoccerHelp?

I read that cPanel can generate your own backup automatically using cPanel API.

I've read that there is also a way to backup to DropBox.

**Comment by dustinschaerer** [comment]
@david-huddleston I've got the new backup systems configured and operational!

The soccerhelp.com server backup with all the databases is currently 1.9 GB.  That includes the whole server system, the soccerhelp account, the soccerhelp website, andall of its databases.  It then gets sent over to S3 at Amazon for safe storage.

It runs nightly - retaining 7 backups, weekly - retaining 4 backups, and monthly retaining 3 backups.

I've checked the bir-2.com backups and they work as expected.
I'll check the SH when it finishes downloading.
